reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/server/JavaServerDriver.java

```diff
@@ -145,7 +145,10 @@ public RequestHandler parse() {
                 ParseMarble pm = new ParseMarble(marble, s);
                 s.onSubscribe(new TestSubscription(pm));
                 new ParseThread(pm).start();
-            }
+            } else {
+                ConsoleUtils.failure("Request response payload " + initialPayload.getK() + " " + initialPayload.getV()
```

Comment:
It isn't very obvious that this will cause the test to fail.  The javadoc says "Logs a failure event".  But more importantly it's a hidden boolean.

This isn't a new issue but I missed it until now.